CMakeLists.txt

```diff
@@ -1881,6 +1881,18 @@ if(GNU_GCC OR LLVM_CLANG)
 endif()
 target_link_libraries(mixxx-lib PRIVATE FpClassify)
 
+# Cpp Core Guideline Support Library (GSL)
+include(FetchContent)
+
+FetchContent_Declare(GSL
```

Comment:
Will this work without internet access during the build? If yes then in which directory are those downloaded archives expected to reside in?

Please compare with how libkeyfinder is integrated. This allows to manually download the archives in advance and run the build without an internet connection.

Otherwise it would break at least the RPM Fusion builds.

Reply:
Mhmmm yeah, this was the recommend integration by MS, I'll look into offline ways to build this.